static/app/components/modals/inviteMembersModal/index.tsx

```diff
@@ -515,7 +515,7 @@ const StatusMessage = styled('div')<{status?: 'success' | 'error'}>`
 
 export const modalCss = css`
   width: 100%;
-  max-width: 800px;
+  max-width: 900px;
```

Comment:
Would be nice to have a set of predefined modal sizes from the theme at some point